### DISCUSSION TITLE
ky equivalent of got.stream ?

### DISCUSSION THREAD

**Discussion by PixeledCode** [sindresorhus/ky#558] [Q&A] [answered]
I am trying to migrate from `got` to `ky` but I am not how to handle the functionality of `got.stream`

remaining code to be migrated:
```
import { createWriteStream } from 'fs';
import { promisify } from 'util';

const pipeline = promisify(Stream.pipeline);
await pipeline(got.stream(url), createWriteStream(tempFile));
```

**Comment by sindresorhus** [comment, marked as answer]
Ky does not currently support streaming, but you could use `fetch` directly:

```js
import {createWriteStream} from 'node:fs';
import {pipeline} from 'node:stream/promises';

const response = await fetch(url);
await pipeline(response.body, createWriteStream(tempFile));
```

**Comment by sholladay** [comment]
Streaming requests and responses are fully supported in the fetch spec and by Ky. However, browser support for streaming requests is somewhat spotty, but it's getting better. See: https://developer.mozilla.org/en-US/docs/Web/API/Request/Request#browser_compatibility
